000-vip-init.php

```diff
@@ -163,6 +163,7 @@
 require_once( __DIR__ . '/lib/utils/class-alerts.php' );
 
 // Load our development and environment helpers
+require_once( __DIR__ . '/vip-helpers/vip-notoptions-mitigation.php' );
```

Comment:
It would be better to split this out to a separate change that can be deployed after the actual file is committed / deployed, to avoid any problems with ordering in the code checkout process.